src/js/helpers/cleanData.js

```diff
@@ -460,23 +459,41 @@ export const clean311 = threeoneone => {
   });
 };
 
-const getDocumentPdfSize = async document => {
+const checkUrl = async url => {
   return await axios({
     method: 'HEAD',
-    url: document.link,
+    url: url,
   })
-    .then(res => {
-      if (res.headers['content-type'] === 'application/pdf') {
-        document.pdfSize = filesize(+res.headers['content-length']).replace(
-          ' ',
-          '',
-        );
-        return;
-      }
-    })
+    .then(res => url)
     .catch(error => null);
 };
 
+const getWorkingDocumentLink = async filename => {
+  // Single source mode, example use case:
+  // If we're on PROD, we should only get prod documents
+  if (process.env.CMS_DOCS !== 'multiple') {
+    return `${process.env.CMS_DOCS}/${filename}`;
+  }
+
+  // Multi source mode, let's do some url checking and get something
+  // that works. example use case:
+  // If we're on STAGING, we want docs imported from PROD to work,
+  // as well as any new docs we added when testing on staging
```

Comment:
Ok. So this allows us to not need to datadump documents as well, correct? But still look at prod documents on staging, but not look at staging documents on prod.

Alternatively, the document saved in the db could include its root url/CMS_DOCS location.

Reply:
Another thought if we keep a CMS_DOCS == 'multiple' flag: could this just be a trigger based on NODE_ENV?

At some point it'd be great to tie environment variables to NODE_ENV and have an organized way to do that. But today is probably not that day.